### PR TITLE
Handler and Patch parts where UsedItem was ItemUsed

### PR DESCRIPTION
**Pull request by RyogenX**
Title say everything

### CODE REVIEW

**Comment by Jesus-QC** [comment]
why

**Comment by RyogenX** [comment]
> why

Just wanted to fix that little error :(

File: Exiled.Example/Example.cs [hunk 1]
```diff
@@ -92,6 +92,7 @@ private void RegisterEvents()
             Exiled.Events.Handlers.Player.Shooting += playerHandler.OnShooting;
             Exiled.Events.Handlers.Player.ReloadingWeapon += playerHandler.OnReloading;
             Exiled.Events.Handlers.Player.ReceivingEffect += playerHandler.OnReceivingEffect;
+            Exiled.Events.Handlers.Player.TriggeringTesla += playerHandler.OnTriggeringTesla;
```
Comment (joker-119) [inline Exiled.Example/Example.cs:95]: You never unregister this event.

Also I don't see the point of any of the changes in Exiled.Example, I feel there's already more than enough code examples for people to get an idea of what to do.

**Review by joker-119 — CHANGES_REQUESTED** [review]

File: Exiled.Example/Example.cs (hunk 1 above)
Comment (RyogenX) [reply Exiled.Example/Example.cs:95]: ups sorry, never mean for this to got to the pr, just the correction of the event, experimenting with pr rn

Comment (RyogenX) [reply Exiled.Example/Example.cs:95]: I don't actually know how to do PR efficienly, If I need to delete the fork and make another one with just the UsedItem correction I can do it.

**Pull request closed without merging**